### PR TITLE
Shanoir-issue#1324 shanoir-uploader filters update

### PR DESCRIPTION
**Pull request by jcomedouteau**

### CODE REVIEW

**Comment by michaelkain** [comment]
ah, I see, please keep the system entry in the pom.xml, as it will not work otherwise on GitHub, as long as the lib is still required; I think you removed it locally to make it work.

File: shanoir-uploader/src/main/java/org/shanoir/uploader/action/ImportFromCsvRunner.java [hunk 1]
```diff
@@ -453,25 +467,64 @@ protected boolean searchField(String searchedElement, String filter) {
...
+		// NB: It is possible to have AND ";" filters OR OR ";;" filters but not both at the same time for the moment.
+
+		if (filter.contains(";;")) {
+			valid = false;
+			filters = filter.split(";;");
+			for (String filterToApply : filters) {
+				if (filterToApply.startsWith("!")) {
+					valid =  valid || !filterWildCard(searchedElement, filterToApply.replaceAll("!", ""));
+				} else {
+					System.err.println(filterToApply + filterWildCard(searchedElement, filterToApply.replaceAll("!", "")));
```
Comment (michaelkain) [inline shanoir-uploader/src/main/java/org/shanoir/uploader/action/ImportFromCsvRunner.java:484]: Hi Jean-Come, would it be okay to use the logger here, that it comes into the su.log please?

**Review by michaelkain — APPROVED** [review]
Hi Jean-Come,
thank you and in general: approved!
Just please fix the pom.xml and the logger issue, and I will merge it.
THX and with kind regards, Michael

**Comment by jcomedouteau** [comment]
It should be good now.

**Review by michaelkain — APPROVED** [review]

**Pull request merged**